### PR TITLE
Fix Repology workflow

### PR DESCRIPTION
**Pull request by Zopolis4**
I hate github actions :(

Anyways, fixed the workflow again by ensuring that crew does not try to take everything as an argument.

### CODE REVIEW

File: .github/workflows/Repology.yml [hunk 1]
```diff
@@ -12,8 +12,8 @@ jobs:
         run: |
             sudo docker pull satmandu/crewbuild:amd64
             sudo docker run -t -v $(pwd):/usr/local/json satmandu/crewbuild:amd64 sudo -i -u chronos /bin/bash -c "
-            crew update
-            ruby ../tools/json.rb
+            crew update &&
+            ruby ../tools/json.rb &&
             cp repology.json /usr/local/json"
```
Comment (satmandu) [inline .github/workflows/Repology.yml:17]: Does this need to be
```
crew update && \
ruby ../tools/json.rb && \
cp repology.json /usr/local/json"
```

Comment (Zopolis4) [reply .github/workflows/Repology.yml:17]: It'll work without them

Comment (uberhacker) [reply .github/workflows/Repology.yml:17]: Just so you know, the `&&` means if any command fails the following commands will not execute.

Comment (Zopolis4) [reply .github/workflows/Repology.yml:17]: In this case I'm just using it to separate the commands so crew doesn't misunderstand. Although, the behaviour of not continuing from a fail is probably desired here.

**Review by satmandu — APPROVED** [review]

**Pull request merged**